src/markdown-pages/add-ons/rook.md

```diff
@@ -10,7 +10,9 @@ addOn: "rook"
 The [Rook](https://rook.io/) add-on creates and manages a Ceph cluster along with a storage class for provisioning PVCs.
 It also runs the Ceph RGW object store to provide an S3-compatible store in the cluster.
 
-By default the cluster uses the filesystem for storage. Each node in the cluster will have a single OSD backed by a directory in `/opt/replicated/rook`.
+By default the cluster uses the filesystem for storage. Each node in the cluster will have a single OSD backed by a directory in `/opt/replicated/rook` as well as a Mon backed by a directory in `/var/lib/rook`.
```

Comment:
This isn't quite true for mons. No more than 3 nodes will have mons at any time.